Source/Core/Common/SysConf.cpp

```diff
@@ -89,9 +90,21 @@ void SysConf::ApplySettingsFromMovie()
   if (!Movie::IsMovieActive())
     return;
 
-  SetData("IPL.LNG", Movie::GetLanguage());
-  SetData("IPL.E60", Movie::IsPAL60());
-  SetData("IPL.PGS", Movie::IsProgressive());
+  auto movie_layer = Config::GetLayer(Config::LayerType::Movie);
+  auto* core = movie_layer->GetOrCreateSection(Config::System::Main, "Core");
+  auto* display = movie_layer->GetOrCreateSection(Config::System::Main, "Display");
+
+  u8 language;
+  core->Get("Language", &language, 0);
```

Comment:
Looking at this code makes me feel there should maybe be a simpler API. Suggestion:

    SetData("IPL.LNG", core->Get<u8>("Language", 0));